src/System.Memory/tests/Memory/MemoryManager.cs

```diff
@@ -28,30 +28,14 @@ public static void MemoryFromMemoryManagerInt()
         }
 
         [Fact]
-        public static void MemoryManagerCtorDefault()
```

Comment:
We should keep this test case as well (just modify to use the protected methods).

Reply:
Never passes `null` into the internal ctor, as it passes `this` so its only a `Debug.Assert` rather than a `throw` - so these tests would never pass.

Reply:
Well... would crash in Debug rather than throwing